statsmodels/datasets/tests/test_utils.py

```diff
@@ -18,8 +18,9 @@ def test_get_rdataset():
         duncan = get_rdataset("Duncan", "car", cache=cur_dir)
         assert_(duncan.from_cache)
 
-# temporarily disabled because some test servers don't have internet
-def t_est_webuse():
+internet_available = check_internet()
+@dec.skipif(internet_available is False)
```

Comment:
I think this can be just `@dec.skipif(not internet_available)`
internet_available is already a boolean